Improve Eq Terminology (#13)
jonasnick/bip-frost-dkg: README.md

TODO Add a more concrete example with lost funds that demonstrates the risk?

While we cannot guarantee in all application scenarios that Eq() terminates and returns, we can typically achieve a weaker guarantee that covers agreement in the successful cases.
Under the assumption that network messages eventually arrive (this is often called an "asynchronous network"), we can guarantee that if *some* honest participant determines the DKG to be successful, then *all* other honest participants determine it to be successful eventually. 

More formally, Eq must fulfill the following properties:
 - Integrity: If some honest participant outputs True, then for every pair of values x and x' input provided by two honest participants, we have x = x'.
 - Conditional Agreement: If some honest participant outputs True and the delivery of messages between honest participants is guaranteed, then all honest participants output True.

> If the protocol has conditional agreement and two honest participants have `x` and `x'` such that `x != x'`, the protocol will not terminate.

Reviewer: s/will not/may not. or phrased differently:
```suggestion
Conditional agreement does *not* guarantee that the protocol terminates if two honest participants have `x` and `x'` such that `x != x'`.
```

Author: Fixed. The "will not" wording should appear in the context of certifying_eq.
